src/oslo_aero_depos.F90

```diff
@@ -717,6 +718,12 @@ subroutine oslo_aero_depos_wet ( lchnk, ncol, psetcols, pmid, pdel, q, t, &
                 jnv = 0  !==> below cloud scavenging coefficients are zero (see above)
              endif
 
+! OS 240207 Increase scavenging efficiency for large soluble particles.
```

Comment:
`git blame` does a good job of showing who made a change and when, we don't really need these inserts.
```suggestion
! Increase scavenging efficiency for large soluble particles.
```